aikau/src/main/resources/alfresco/renderers/css/Thumbnail.css

```diff
@@ -1,48 +1,52 @@
 .alfresco-renderers-Thumbnail {
-   width: 100px;
+   width: @thumbnail-width;
+   height: @thumbnail-height;
    position: relative; /* This is required for absolutely positioned overlays */
-}
 
-.alfresco-renderers-Thumbnail span.link
-{
-   background: url(./images/link-32.png) no-repeat;
-   height: 32px;
-   width: 32px;
-   position: absolute;
-}
+   .alfresco-renderers-Thumbnail__frame {
+      display: inline-block;
+      height: @thumbnail-height;
+      width: @thumbnail-width;
+      line-height: @thumbnail-height;
+      text-align: center;
+   }
 
-.alfresco-renderers-Thumbnail span.link.hidden
-{
-   display: none;
-}
+   .alfresco-renderers-Thumbnail__image {
+      cursor: pointer;
+      max-height: ~'calc(100% - @thumbnail-margin - @thumbnail-margin - @thumbnail-border-width)';
```

Comment:
It's the difference between "looks bad on IE8" and "fundamentally doesn't work on IE8".... we can live with looking bad (after all, that'll probably be the least of the users problems).

Having said that, looking at the code I wonder if we can do the calculation entirely in LESS? After all the 100% here is simply going to be the thumbnail height. I'll look at that again.